packages/compile-service/src/index.ts

```diff
@@ -32,14 +37,41 @@ export type DbDeployQueue = {
     if (request) {
       logger.info(`Received ${request.databaseName} to queue`);
       try {
-        const response = await zkAppCompiler.compileAndCreateUnsignTx(request);
+        let transaction: UnsignedTransaction;
+
+        if (request.transactionType === "deploy") {
```

Comment:
I think you can make sure it by using
```ts
if (request.transactionType === "deploy") ...
if (request.transactionType === "rollup") ...
```

When ts become js in runtime, you can pass some string that not "deploy" and it will go to the else

Reply:
done

Reply:
Are you push it yet? I still see the old one